xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-macro/xwiki-platform-notifications-macro-ui/src/main/resources/XWiki/Notifications/Code/Macro/NotificationsMacro.xml

```diff
@@ -914,7 +914,7 @@ require(['jquery', 'XWikiAsyncNotificationsMacro', 'xwiki-events-bridge'], funct
       <name>CSS</name>
     </property>
     <property>
-      <parse>1</parse>
+      <parse>0</parse>
```

Comment:
This seems wrong, it means that the Velocity code above won't be evaluated anymore.